README.md

```diff
@@ -322,33 +322,27 @@ have an understanding of the topic yourself, and know what questions to ask.
 
 ### Compensation and equity
 
-🄳 **Compensation** is what you get in exchange for working for a company, including cash
-and non-cash pay.
+🄳 **Compensation** is what you get in exchange for working for a company, usually including
```

Comment:
Generally OK but two imprecisions:
- Elide "usually" as we're defining compensation. The definition itself isn't "usually", even if you don't usually get it.
- This now makes it sound like non-cash pay is *only* benefits. Equity is also non-cash pay (by most people's definitions).

Would rewrite to

🄳 **Compensation** is what you get in exchange for working for a company. This includes cash pay (your salary and bonus, if you get one) and non-cash pay, including [benefits](https://en.wikipedia.org/wiki/Employee_benefits#United_States) like ...

Reply:
Thank you! Excellent!